card/spec/lib/card/reference_spec.rb

```diff
@@ -197,8 +197,8 @@
   end
 
   it "pickup new inclusions on create" do
-    @l = Card.create! :name=>"woof", :content=>"{{Lewdog}}"  # no Lewdog card yet...
-    @e = Card.new(:name=>"Lewdog", :content=>"grrr")              # now there is
+    @l = Card.create! name: "woof", content: "{{Lewdog}}"  # no Lewdog card yet...
+    @e = Card.new(name: "Lewdog", content: "grrr")              # now there is
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.